sim/battle-queue.ts

```diff
@@ -177,8 +177,9 @@ export class BattleQueue {
 				runSwitch: 101,
 				runPrimal: 102,
 				switch: 103,
-				megaEvo: 104,
-				runDynamax: 105,
+				rotate: 104,
+				megaEvo: 105,
+				runDynamax: 106,
```

Comment:
Is this definitely the order that things happen in-game?

Reply:
Woble said he'd do some reasearch, but rotations definitely happen after hard-switches.

Reply:
Ah, you're waiting on research? Fair enough.